vm/systemSmartContracts/governance.go

```diff
@@ -946,8 +979,8 @@ func (g *governanceContract) saveNewVoteData(voter []byte, voteData *VoteData, p
 	return nil
 }
 
-// saveVoteData saves the provided vote data into the storage
-func (g *governanceContract) saveVoteData(voter []byte, voteData *VoteData, proposalReference []byte) error {
+// storeVoteSet saves the provided vote data into the storage
+func (g *governanceContract) storeVoteSet(voter []byte, voteData *VoteSet, proposalReference []byte) error {
```

Comment:
we have both `saveVoteSet` and `storeVoteSet` which basically mean the same thing bu apparently do something different.

Reply:
Yeah, `storeVoteSet` just writes the vote set into the storage. `saveVoteSet` also updates the proposal it is linked to. I am thinking of a better naming here, if you have any suggestion is welcome